code/modules/projectiles/guns/energy/special.dm

```diff
@@ -341,8 +341,8 @@
 	var/emagged = 0			//ups the temperature cap from 500 to 1000, targets hit by beams over 500 Kelvin will burst into flames
 	var/dat = ""
 
-/obj/item/gun/energy/temperature/New()
-	..()
+/obj/item/gun/energy/temperature/Initialize(mapload, ...)
```

Comment:
What is the use of `...` in this proc?

Reply:
I believe that is a holder for arguments or such. Using `args` you can get them. I think this is what my intellisense generated when I selected Initialize